### ISSUE TITLE
Stop Button on Plotting

### ISSUE THREAD

**Issue by Drhicom** [madMAx43v3r/mmx-node#79]
Would it be good to have some way to cancel or STOP the plotting process with a STOP button?

**Comment by limsandy** [comment]
Ctrl + C once will stop the ploter after the last plot is done.
Ctrl + C multiple times will kill the plotter.

**Comment by Drhicom** [comment]
Just like he did in his other software
![image](https://user-images.githubusercontent.com/81637574/176982014-d4d6ea2d-1f9e-4845-8fbe-bd0f2c22b12c.png)

**Comment by limsandy** [comment]
If you're talking about the Windows version, Stotiks is managing that. But I don't think he ever made one with a GUI. So I don't know what program is on your screenshot. Can you show the setting tab?

I searched and found nothing that match "Plot Manager". /shrug

**Comment by Drhicom** [comment]
The file downloaded in 10/18/2021 (PlotManager.zip) In the readme.txt file at close to the bottom is
Check my github for latest updates:
https://github.com/stotiks/chia-plotter/releases

Build based on madMAx43v3r source code, check his github:
https://github.com/madMAx43v3r/chia-plotter
He is a legend!

**Comment by Drhicom** [comment]
Looking more
https://github.com/stotiks/chia-plotter/releases/tag/v0.1.5

Here is your answer, hope it helps, If you download Plotmanager.zip you will see.

**Comment by limsandy** [comment]
Ahhh I see.... I didn't know he made a GUI version. Anyway, the GUI seems to be using an older version of the plotter.

I suggest if you're going to use the GUI, download the latest version of the plotter from
https://github.com/MMX-World/mmx-plotter/releases
Also maintained by Stotiks. And then rename the new plotter the same filename as in Plot Manager.
Try it, it should work. ;)

**Comment by limsandy** [comment]
If you haven't, come join our Discord channel. Sometimes you can talk to Max, and several other great people there. The link can be found in this Github page. ;)

**Comment by Drhicom** [comment]
Thanks, I already joined discord MMX group. Now getting back to the original post, it would be nice to have a stop button. I'm sure Stotiks could sneak it in there since they work together, correct?

**Comment by Jacek-ghub** [comment]
I would like to second @Drhicom as far as updating that plot manager.  The #1 chia related program sought after the initial Chia release was some kind of plot manager, and several came online at some point.  However, scumbags took advantage of that need, and started to push malware targeting chia installs / stealing XCHes for those more gullible or less tech oriented.  If that plotter will be updated / integrated with Win release, that hole will be plugged day one, also lowering the barrier of entry on the plotter side.  (I am using CLI for plotting, so I am not advocating for myself).

One can imagine that new features could be added to the plotter (multiple drives as destinations, parallel destinations in case plotter is faster than destination, adding / removing drives on the fly without restarting manager, ...).  Seeing how quickly Andrei is addressing issues with Win build, I have no doubt that he can do a lot with it (given he has time for it).

Also, I would think that telling people to join discord and do everything there is not really the best approach.  Github has some advantages (tickets, in addition to issues, we could have Discussions page (items that are not really bugs), labels (bug, suggestion, support, ...).  Currently discord is a treasure trove.  However, after a couple of days all that good stuff goes into oblivion, plus it is hard to have a decent exchange when multiple issues start to mingle.  In addition, those sideline chats and animated gif images are not really helping.

By the way, I have just purchased mmxforum.com domain.  If you want, I can xfr it to you (or rather Max).  If you would like to be admin, I can try to install Discourse on it.  I like chiaforum.com, and actually learned a lot there.  At least for me, such forum also has some benefits.

**Comment by limsandy** [comment]
I'll leave Andrei to address this issue himself if he decides to.... I have sent the link to this discussion to him but we haven't heard from him here. He has, however, been active on Discord #plotting and he said the plot manager is not very efficient because he had to dedicate 1 CPU core for console output. Having said that, I don't think there will be (major) further development on the plotter, both Max's Linux and Stotils' Windows version.

BTW, you can use the search feature on Discord for old topics that has been discussed before. If you decide to stay here to wait for answers, that's your choice. I'll be more active on Discord, in fact, I consider this topic closed and I'm heading over to Discord now. :)

**Comment by Drhicom** [comment]
So this is going to be added to v24? And did Andrei ask you to close this?

**Comment by Drhicom** [comment]
Thank you for putting this option in.
